components/vf-comments/vf-comments.hbs

```diff
@@ -14,7 +14,7 @@
         <p>For me, any communications strategy should be deeply rooted in the organisational strategy. And yet, your closing question reminded me how the discussions we lead about communications can often result in much broader discussions on organisational goals, ambitions and approaches.</p>
       </div>
       <button class="vf-button vf-button--outline--primary | vf-comments__reply">Reply</button>
-      <a href="#" class="vf-button vf-button--outline--tertiary | vf-comments__reply">Optional action</a>
+      <a href="#" class="vf-button vf-button--outline--tertiary | vf-comments__action">Optional action</a>
```

Comment:
do you think this would be an action that takes you to a different page? 

I guess _really_ the reply button would be going to another page on most things as most things won't be a SPA ?

Reply:
Yeah. Think in reality most would be new pages... Thinking things like: "edit" "flag as spam" ... I suppose we might eventually need more designs on those, but this felt like a reasonable first step/thought.